### ISSUE TITLE
Why not data entry in grid cells like Excel?

### ISSUE THREAD

**Issue by sky5walk** [sqlitebrowser/sqlitebrowser#1669]
I do not generally edit tables manually, but recently I noticed that entering data into cells is really quite frustrating. I type my number or text and hit an arrow key [right arrow] or an [Enter] key and the data is NOT accepted.
I do see a dropdown box appear and that may be of assistance for textual entry, but I am simply entering numbers. How can I disable the "Suggestion List" and force acceptance of my data on the following keys?
```
[any arrow key] <-- Accepts data and moves to next cell according to direction of arrow.
[Enter]         <-- Accepts data and moves down 1 cell if next row exists.

```
----------------
#### Does this feature exist in another product or project? Please provide a link.
This is the standard method of entry in Excel and much more fluid than DB4S.

**Comment by mgrojo** [comment]
You can disable the drop-down setting _Preferences > Data Browser > Row count threshold for completion_  to the value 0. Then you'll get back to the previous behaviour. But that isn't either what Excel does. You can use Tab, though, for the same effect than Left arrow, or Enter and arrow, for any other direction.

**Comment by chrisjlocke** [comment]
> You can use Tab, though, for the same effect than Left arrow, or Enter and arrow, for any other direction.

When entering data, and the selection dropdown has dropped, pressing <kbd>Tab</kbd> has no effect (on Windows, anyhow).  For me, the selection is useful, but also annoying and frustrating, where a selection has to be selected and approved to continue.  Its not exactly how Excel works, which is what I'm used to.

**Comment by sky5walk** [comment]
Yes, I tried dropping Row count threshold = 0 and it is a little better.
But, the Excel behavior is kinda universally applied to any grid I've used until now. 
In fact, DB4S responds to [F2] as a signal to edit the current cell. It appears, we just need the rest of the key actions implemented. [Enter], [Arrows], [Tab] key, etc.
